core/src/main/scala/com/github/mjakubowski84/parquet4s/ParquetWriter.scala

```diff
@@ -108,6 +155,18 @@ object ParquetWriter  {
         writer.close()
       }
     }
+
+    override def writeS(outputFile: OutputFile, data: Iterable[T], options: Options): Unit = {
```

Comment:
instead of `writeS` I would create a type class for Sinks e.g.:
```
def write[T: ParquetWriter, S: Sink](output: S, data: Iterable[T], options = options: ParquetWriter.Options = ParquetWriter.Options())
```

Then I would provide `Sink` implementations for both `File` and `OutputStream`